cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json

```diff
@@ -28,6 +28,7 @@
     "transforms": [],
     "instances": 1
   },
-  "description": "Realtime updates from Amazon Simple Queue Service to HBase."
+  "description": "Real-time updates from Amazon Simple Queue Service into an HBase table",
```

Comment:
We decided to either use "real time" or "real-time", defaulting to "real time" and using the hyphenated form to avoid ambiguity. In this case, "real-time" makes sense.